sh_localized_damage.lua

```diff
@@ -0,0 +1,70 @@
+local PLUGIN = PLUGIN
+PLUGIN.name = "Localized damage"
+PLUGIN.author = "Subleader"
+PLUGIN.description = "Damage are different depending on which limb is being hurt."
+
+ix.config.Add("localizedDamage", true, "Activate the Localized Damage.", nil, {
+	category = "Localized damage"
+})
+
+ix.config.Add("legBreakDuration", 10, "Set the time that a leg remains broken.", nil, {
+	data = {min = 0, max = 300.0, decimals = 1},
+	category = "Localized damage"
+})
+
+ix.config.Add("HeadScaleDamage", 3, "How much should head damage be scaled by?", nil, {
+	data = {min = 0, max = 10.0, decimals = 1},
+	category = "Localized damage"
+})
+
+ix.config.Add("ArmsScaleDamage", 0.4, "How much should arms damage be scaled by?", nil, {
+	data = {min = 0, max = 10.0, decimals = 1},
+	category = "Localized damage"
+})
+
+ix.config.Add("LegsScaleDamage", 0.5, "How much should legs damage be scaled by?", nil, {
+	data = {min = 0, max = 10.0, decimals = 1},
+	category = "Localized damage"
+})
+
+ix.config.Add("StomachScaleDamage", 0.8, "How much should stomach damage be scaled by?", nil, {
+	data = {min = 0, max = 10.0, decimals = 1},
+	category = "Localized damage"
+})
+
+function LegBreak(ply,duration)
+	if !RUNSPEED and !WALKSPEED then
+		RUNSPEED = ply:GetRunSpeed()
+		WALKSPEED = ply:GetWalkSpeed()
+	end
+
+	if !ply.legshot then
+		ply.legshot = true
+		ply:SetRunSpeed(RUNSPEED/3)
+		ply:SetWalkSpeed(WALKSPEED/3)
+		timer.Simple(duration,function() ply:SetRunSpeed(RUNSPEED) ply:SetWalkSpeed(WALKSPEED) ply.legshot = false end)
+	end
+end
+
+function PLUGIN:ScalePlayerDamage( ply, hitgroup, dmginfo )

```

Comment:
You should use `client` instead of `ply` as the player argument.
Spacing is not needed for the brackets in function arguments.